azure-pipelines.yml

```
@@ -637,17 +637,25 @@ jobs:
   strategy:
     matrix:
       Red Hat Universal Base Image 8:
-        image: ubi8
-        tag: 8.4
+        dockerfile: ubi
+        image: registry.access.redhat.com/ubi8/ubi:8.4
         artifact: rpm-ubi8
+        python_package: python39
+      Red Hat Universal Base Image 9:
+        dockerfile: ubi
+        image: registry.access.redhat.com/ubi9/ubi:9.0.0
```

Can we unpin the last version?

```suggestion
        image: registry.access.redhat.com/ubi9/ubi:9.0
```

---

9.0 returns 404.

---

Indeed I got failure:

```
> docker run -it --rm registry.access.redhat.com/ubi9/ubi:9.0
Unable to find image 'registry.access.redhat.com/ubi9/ubi:9.0' locally
docker: Error response from daemon: error parsing HTTP 404 response body: invalid character 'N' looking for beginning of value: "Not found\n".
See 'docker run --help'.
```

To much inconsistency! 😣